scripts/data_collector/pit/test_pit.py

```diff
@@ -9,15 +11,22 @@ class TestPIT(unittest.TestCase):
     """
     NOTE!!!!!!
     The assert of this test assumes that users follows the cmd below and only download 2 stock.
-    `python collector.py download_data --source_dir ./csv_pit --start 2000-01-01 --end 2020-01-01 --interval quarterly --symbol_flt_regx "^(600519|000725).*"`
+    1. `python scripts/get_data.py qlib_data --target_dir ~/.qlib/qlib_data/cn_data --region cn`
+    2. `python scripts/data_collector/pit/collector.py download_data --source_dir ~/.qlib/stock_data/source/pit --start 2000-01-01 --end 2020-01-01 --interval quarterly --symbol_regex "^(600519|000725).*"`
+    3. `python scripts/data_collector/pit/collector.py normalize_data --interval quarterly --source_dir ~/.qlib/stock_data/source/pit --normalize_dir ~/.qlib/stock_data/source/pit_normalized`
+    4. `python scripts/dump_pit.py dump --csv_path ~/.qlib/stock_data/source/pit_normalized --qlib_dir ~/.qlib/qlib_data/cn_data --interval quarterly`
     """
 
     def setUp(self):
         # qlib.init(kernels=1)  # NOTE: set kernel to 1 to make it debug easier
-        qlib.init()  # NOTE: set kernel to 1 to make it debug easier
+        qlib.init()
 
     def to_str(self, obj):
-        return "".join(str(obj).split())
+        if isinstance(obj, pd.DataFrame):
```

Comment:
I prefer not to check all the data.  It looks a little lengthy.
The short version of str(obj) will be enough for most cases.

Reply:
in `test_expr` case the str(data) result like:

![image](https://user-images.githubusercontent.com/32626585/158986141-69d4c86f-7f57-4671-8d1f-cde6cdfefafe.png)

The middle columns are all omitted.